### DISCUSSION TITLE
2023-07-10 Bundle Release

### DISCUSSION THREAD

**Discussion by cathay4t** [rust-netlink/.github#6] [Announcements]
## Highlights

 * Continue the work on removing all reexports.
 * Removed `netlink-packet-core::NetlinkPayload::Ack`, replaced by
   `NetlinkPayload::Error` with `ErrorMessage.code` is set to None.

## Versioning
 * netlink-packet-core 0.5.0 -> 0.7.0
 * netlink-proto 0.11.1 -> 0.11.2
 * netlink-packet-xfrm 0.1.1 -> 0.3.1
 * netlink-packet-generic 0.3.2 -> 0.3.3
 * genetlink 0.2.4 -> 0.2.5
 * mptcp-pm 0.1.2 -> 0.1.3
 * netlink-packet-route 0.15.0 -> 0.17.0
 * rtnetlink 0.12.0 -> 0.13.0
 * netlink-packet-audit 0.5.0 -> 0.5.1
 * netlink-packet-sock-diag 0.4.1 -> 0.4.2
 * audit 0.7.2 -> 0.7.3
 * netlink-packet-wireguard 0.2.2 -> 0.2.3
 * netlink-packet-netfilter 0.1.0 -> 0.2.0
 * wl-nl80211 0.1.1 -> 0.1.2
 * ethtool 0.2.4 -> 0.2.5
 * xfrmnetlink 0.2.0 -> 0.2.1

**Comment by stappersg** [comment]
And while this _bundle release_ is _work in progress_ there will be breakages. In https://github.com/rust-netlink/genetlink/issues/7 is said:

> I am in middle of bundle release of rust-netlink crates. Please wait a while.

Regarding the _Please wait a while_:   What to expect?  Hours?  Days?   Weeks?  Months?

**Reply by cathay4t** [reply]
The bundle release normally takes up to two days.

**Comment by cathay4t** [comment]
It should be fixed.
